src/Spout/Reader/Helper/XLSX/SharedStringsHelper.php

```diff
@@ -34,6 +34,9 @@ class SharedStringsHelper
     /** Value to use to escape the line feed character ("\n") */
     const ESCAPED_LINE_FEED_CHARACTER = '_x000A_';
 
+    /** This will increase your memory usage but can improve your execution time */
+    static public $KEEP_ALL_STRINGS_IN_MEMORY = false;
```

Comment:
Mhh I found this problem in a document with ~130k shared strings. The problem is that the strings aren't in the 'right' order. In some rows the helper need to reread the cache files again and again and agian.